src/main/java/teammates/common/util/Const.java

```diff
@@ -701,6 +701,9 @@ private Const() {
         public static final String FEEDBACK_SESSION_TIMEZONE = "timezone";
         public static final String FEEDBACK_SESSION_GRACEPERIOD = "graceperiod";
         public static final String FEEDBACK_SESSION_TYPE = "fstype";
```

Comment:
do you think we should avoid the confusing use of the name `FEEDBACK_SESSION_TYPE`?

Reply:
I think this arose from the fact that the original intention of feedback session type is also to encapsulate different feedback session templates (see `FeedbackSessionType.java`'s comments). If we agree to separate feedback session types from feedback session templates then yes, we should rename it

Reply:
emmmm i dont think the intention of feedback session type is to encapsulate different templates tho...(I used to think it that way but after reading the code at `FeedbackSessionsLogic` i changed my mind. Moreover, this method is only used in `FeedbackSessionStatsPageAction`, which is mapped to `INSTRUCTOR_FEEDBACK_STATS_PAGE`). To me it seems to be specifying the "kind" of the feedback session e.g. is the feedback session only for the creator of the session himself (i.e PRIVATE) or is it for the students(i.e. STANDARD)...The comments to me seem to be specifying other possible "kinds" of the feedback session e.g. open to all instructors...